packages/ui/src/apis/utils.ts

```diff
@@ -15,32 +15,32 @@ import {
 
 export type PasscodeChannel = 'sms' | 'email';
 
-export const getSendPasscodeApi = (type: UserFlow, channel: PasscodeChannel) => {
-  if (type === 'sign-in' && channel === 'email') {
+export const getSendPasscodeApi = (type: UserFlow, method: PasscodeChannel) => {
```

Comment:
seems that always return `sendSignInEmailPasscode` when `method` is `email` and return `sendRegisterSMSPasscode` when `method` is `sms`.

could we simplify it? (maybe do it in another PR)